drakrun/drakrun/draksetup.py

```diff
@@ -75,6 +76,13 @@ def ensure_zfs(ctx, param, value):
     return value
 
 
+def check_root():
+    if os.getuid() != 0:
+        logging.exception("Please run the command as root")
```

Comment:
This is not related to an exception, so please use:

```suggestion
        logging.error("Please run the command as root")
```